.github/workflows/release.yml

```diff
@@ -190,16 +190,7 @@ jobs:
           fi
 
       - name: Install universal-sierra-compiler
-        shell: bash
```

Comment:
From what I've seen, a support for windows has been added to the gh action - that is cool. Do you happen to know if there any plans to also include this support to install.sh script?

Reply:
Sure, I could do that later